.c8rc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "report-dir": "./coverage",
+    "reporter": ["lcov"],
+    "all": true,
+    "include": ["src/**", "dist/src/**"],
```

Comment:
should we check the lint rules in plugins/ as well?

Reply:
Sure, let's add that later.